src/routes/(app)/checkout/+page.svelte

```diff
@@ -33,8 +33,9 @@
 	let onLocation = data.defaultOnLocation;
 	$: offerDeliveryFees = onLocation;
 	let addDiscount = false;
-	let discountAmount = 0;
-	let discountType: DiscountType;
+	let offerOrder = false;
+	$: discountAmount = offerOrder ? 100 : 0;
+	$: discountType = (offerOrder ? 'percentage' : 'fiat') as DiscountType;
```

Comment:
I think it may have weird behavior when interacting with the controls. You can do something like this:

```suggestion
	$: if (offerOrder) {
		discountType = 'percentage';
		discountAmount = 100;
	}
	$: if (discountType !== 'percentage' || discountAmount !== 100) {
		offerOrder = false;;
	}
```